### DISCUSSION TITLE
Storing animation as video file

### DISCUSSION THREAD

**Discussion by ItayVortex** [marcomusy/vedo#581] [Q&A] [answered]
Using for example the animation from example density4d, is it possible to store it as a video file?

**Comment by marcomusy** [comment, marked as answer]
yes - just use 
```python
video = Video("movie.mp4", duration=6, backend='ffmpeg') # backend='opencv'
# ...loop
    video.addFrame()
video.close()                     
```
see example: https://github.com/marcomusy/vedo/blob/master/examples/other/makeVideo.py
